ddl/ddl_api.go

```diff
@@ -3096,7 +3099,9 @@ func SetDirectResourceGroupUnit(resourceGroupSettings *model.ResourceGroupSettin
 // SetDirectResourceGroupRunawayOption tries to set runaway part of the ResourceGroupSettings.
 func SetDirectResourceGroupRunawayOption(resourceGroupSettings *model.ResourceGroupSettings, typ ast.RunawayOptionType, stringVal string, intVal int32) error {
 	if resourceGroupSettings.Runaway == nil {
-		resourceGroupSettings.Runaway = &model.ResourceGroupRunawaySettings{}
+		resourceGroupSettings.Runaway = &model.ResourceGroupRunawaySettings{
+			Action: model.RunawayNoneAction,
```

Comment:
Here are new modifications and add test. @glorv @nolouch @Connor1996

Reply:
Seems this default value is useless